community/community.cpp

```diff
@@ -299,6 +320,10 @@ void cambiatus::upsertaction(std::uint64_t action_id, std::uint64_t objective_id
     eosio::check(verifications >= 3 && ((verifications & 1) != 0), "You need at least three validators and it must be an odd number");
   }
 
+  // Verify proofs parameters
```

Comment:
We can keep the same pattern as before:

```suggestion
  // Validate proofs parameters
```